scripts/tasks/sandbox-parts.ts

```diff
@@ -42,6 +42,7 @@ import { workspacePath } from '../utils/workspace';
 import { babelParse } from '../../code/lib/csf-tools/src/babelParse';
 import { CODE_DIRECTORY, REPROS_DIRECTORY } from '../utils/constants';
 import type { TemplateKey } from '../../code/lib/cli/src/sandbox-templates';
+import { isFunction } from 'lodash';
```

Comment:
Can we get around not using lodash?

Reply:
It is a dependency in `scripts/package.json`. What's the reason you would like to avoid utility helpers from lodash?